facets_overview/components/facets_overview_table/facets-overview-table.ts

```diff
@@ -156,12 +157,23 @@ Polymer({
     return dataModel.getFeature(feature.getName()!, dataset.getName()!);
   },
   _getChartData: function(
-      dataModel: OverviewDataModel, feature: FeatureNameStatistics):
+      dataModel: OverviewDataModel, feature: FeatureNameStatistics,
+      datasetCheckboxes: boolean[]):
       utils.HistogramForDataset[] {
         if (!dataModel || !feature) {
           return [];
         }
-        return dataModel.getDatasetHistogramsForFeature(feature.getName()!);
+        let histograms = dataModel.getDatasetHistogramsForFeature(feature.getName()!);
+        const datasetNames = dataModel.getDatasetNames();
+        console.log(histograms);
+        histograms = histograms.filter(histogram => {
+          if (datasetCheckboxes == null) {
```

Comment:
This check could occur outside of the filter since it doesn't depend on any values inside the loop.

Reply:
done